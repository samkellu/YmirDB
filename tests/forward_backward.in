set a
set b a
set c b a
set d c b a
set e d c b a
set f e d c b a
set g f e d c b a
backward a
backward b
backward c
backward d
backward e
backward f
forward g
forward a
forward b
forward c
forward d
forward e
forward f
forward g
bye
