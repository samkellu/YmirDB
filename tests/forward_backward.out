> ok

> ok

> ok

> ok

> ok

> ok

> ok

> b, c, d, e, f, g

> c, d, e, f, g

> d, e, f, g

> e, f, g

> f, g

> g

> a, b, c, d, e, f

> nil

> a

> a, b

> a, b, c

> a, b, c, d

> a, b, c, d, e

> a, b, c, d, e, f

> bye
